### ISSUE TITLE
ERROR: could not find output plugin output_zmqserver.so

### ISSUE THREAD

**Issue by UnderOverCovered** [jacksonliam/mjpg-streamer#216]
Hello everyone, I am getting an error while opening mjpg-stream with zmq output. So what I got:
`ERROR: could not find output plugin output_zmqserver.so
       Perhaps you want to adjust the search path with:
       # export LD_LIBRARY_PATH=/path/to/plugin/folder
       dlopen: output_zmqserver.so: cannot open shared object file: No such file or directory`
What I did:
`export LD_LIBRARY_PATH=/usr/src/mjpg-streamer/mjpg-streamer-experimental/plugins/output_zmqserver`
Also I installed this packages: libzmq3-dev, libprotobuf-c-dev libprotobuf-c1 libprotoc10 protobuf-c-compiler.
Can you help me please with this issue?
Thank you in advance for your responses. (Sorry for possible text mistakes, English is my second language)

**Comment by UnderOverCovered** [comment]
What about output_zmqserver? Why I can't run it? I am getting an error:
`Perhaps you want to adjust the search path with:
       # export LD_LIBRARY_PATH=/path/to/plugin/folder
       dlopen: output_zmqserver.so: cannot open shared object file: No such file or directory
`
I tried to do this:
`export LD_LIBRARY_PATH=/usr/src/mjpg-streamer/mjpg-streamer-experimental/plugins/output_zmqserver
`
Also I installed this packages: libzmq3-dev,  libprotobuf-c-dev libprotobuf-c1 libprotoc10 protobuf-c-compiler

**Comment by jacksonliam** [comment]
You want to set the LD library path to the location of the .so files. This is usually the mjpg-streamer-experimental directory.

Do you have an output_zmqserver.so file anywhere?

If you installed the dependencies after running make the first time then do a `make distclean` to clear the cmake cache and try building again.

If it still doesn't work after that, post the full output of the build.

**Comment by UnderOverCovered** [comment]
Hello! Thank you very much Jacksonliam! Yeah, I hadn't the output_zmqserver.so file in experimental directory. So I deleted the mjpg-streamer. Then after git download, I ran:
`cd mjpg-streamer-experimental`
`make distclean`
`make`
`sudo make install`
And yes this .so file in the folder. So, I ran mjpg-streamer with output_zmqserver - h and that works.
Thank you very much! Your streamer is the best!
